### PR TITLE
Add mljar-scikit-plot

### PR DESCRIPTION
**Pull request by pplonski**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by pplonski** [comment]
Hi @conda-forge/staged-recipes,

Please could you merge this package, my other packages https://anaconda.org/conda-forge/mljar-supervised depends on this one

File: recipes/mljar-scikit-plot/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  sha256: e9a6a6c6fdfa277a0ced22872350336556d3c8022f3618ad0c7c0ed236069746
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  noarch: python
+  number: 0
+
+requirements:
+  host:
+    - python >=3.9
```
Comment (ocefpaf) [inline recipes/mljar-scikit-plot/meta.yaml:19]: The package classifiers say 3.8 but there is not a python_requires info.

File: recipes/mljar-scikit-plot/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: e9a6a6c6fdfa277a0ced22872350336556d3c8022f3618ad0c7c0ed236069746
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
```
Comment (ocefpaf) [inline recipes/mljar-scikit-plot/meta.yaml:13]:
```suggestion
  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
```

File: recipes/mljar-scikit-plot/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  run:
+    - python >=3.9
+    - matplotlib-base >=1.4.0
+    - scikit-learn >=1.1.0
+    - scipy >=1.11.4
+    - joblib >=0.10
+
+test:
+  imports:
+    - scikitplot
```
Comment (ocefpaf) [inline recipes/mljar-scikit-plot/meta.yaml:30]:
```suggestion
test:
  imports:
    - scikitplot
  commands:
    - pip check
  requires:
    - pip
```

File: recipes/mljar-scikit-plot/LICENSE [hunk 4]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2018 MLJAR Sp. z o.o.
```
[the remark this answers is not in the document]
Comment (pplonski) [reply recipes/mljar-scikit-plot/LICENSE]: Thank you. License file remove.

File: recipes/mljar-scikit-plot/meta.yaml (hunk 1 above)
Comment (pplonski) [reply recipes/mljar-scikit-plot/meta.yaml:19]: I updated the package https://github.com/mljar/mljar-scikit-plot/blob/master/setup.py

File: recipes/mljar-scikit-plot/LICENSE (hunk 4 above)
[the remark this answers is not in the document]
Comment (ocefpaf) [reply recipes/mljar-scikit-plot/LICENSE:3]: You should remove it if it is the same as the one in the tarball. Conda will pick up the one in the source code accordingly. No need to include this file.

File: recipes/mljar-scikit-plot/meta.yaml (hunk 1 above)
Comment (ocefpaf) [reply recipes/mljar-scikit-plot/meta.yaml:19]: NP. Consider using the `python_requires` metadata. Classifiers don't mean anything to a package.

**Comment by pplonski** [comment]
Thank you @ocefpaf, I've applied suggested changes.

File: recipes/mljar-scikit-plot/meta.yaml (hunk 1 above)
Comment (pplonski) [reply recipes/mljar-scikit-plot/meta.yaml:19]: Thanks! :1st_place_medal:  Fixed.

File: recipes/mljar-scikit-plot/LICENSE (hunk 4 above)
[the remark this answers is not in the document]
Comment (pplonski) [reply recipes/mljar-scikit-plot/LICENSE]: OK, removed.

**Comment by pplonski** [comment]
Thank you!

**Pull request merged**